setup.cfg

```diff
@@ -49,6 +49,8 @@ doc =
     sphinx_gallery
     pactools
     nibabel
+    pooch
+    tqdm
```

Comment:
I think installing `mne` from master should already install this dependency no?
Seeing here: https://github.com/mne-tools/mne-python/blob/main/requirements.txt

Reply:
no it doesn't not. It's considered an optional dep of mne that is only required for fetching data. I know it's not perfect but...

Reply:
No, I think it still does not install. See https://github.com/mne-tools/mne-python/blob/61608aa4038a888d1c1f7c9dd5bb1a02203719de/setup.py#L53

Reply:
The way `mne` handle dependencies seems to make no sense but ok, I see why I did not understood :sweat_smile:

```
    hard_dependencies = ('numpy', 'scipy')
    install_requires = list()
    with open('requirements.txt', 'r') as fid:
        for line in fid:
            req = line.strip()
            for hard_dep in hard_dependencies:
                if req.startswith(hard_dep):
                    install_requires.append(req)
```
Let's merge once it is green then !